### ISSUE TITLE
How to get the value for read-only object  (i.e. the table that are dynamically getting created by the protocol)) 

### ISSUE THREAD

**Issue by santhoshmaddikera** [vlvassilev/yuma123#63]
Hi,

As, we know that to get the value for read only object we need to add the virtual node and register the get CB. For registering CB we use this API (agt_make_virtual_leaf and val_add_child). But this API needs the parent object where we want to add,

Suppose we are having entry 1 than we are adding to it But if in a table all are either read-only object or not-accessible(as per MIB) i.e we are not able to identify with which entry we need to add the get CB.
So in this scenario how to get the value from the device.

Any suggestion on how we can add the GET CB in this scenario.
Basically we are facing this issue for stats table where index of the table are not configured by the user in this case we are not able to register the get CB and we are not getting the sget value for those table. 

Please suggest your opinion on this issue.
Thanks
